javascriptv3/example_code/libs/logger.js

```diff
@@ -21,17 +23,20 @@ export class Logger {
     if (!message) {
```

Comment:
The comment and method name are now wrong

Reply:
Done